File: SampleProject-API-C#/Model/PersonalInfo.cs
namespace SampleProject.Model
{
    public class PersonalInfo {
        public PersonalInfo(string userId)
        {
            this.UserId = userId;
        }
        
        public string UserId { get; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public bool IsEmployed { get; set; }
    }
}

File: SampleProject-Android/app/src/main/java/example/com/sampleproject/models/PersonalInfo.java
package example.com.sampleproject.models;

/**
 * Created by User-01 on 12/24/2016.
 */

public class PersonalInfo {
    private String userId;
    private String firstName;
    private String lastName;
    private String email;

    public String getUserId() {
        return userId;
    }

    public void setUserId(String id) {
        this.userId = id;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    // TODO: 1/8/2017 transfer this part later
    //region "This part will be transfered later"
    private boolean isEmployed;

    public boolean isEmployed() {
        return isEmployed;
    }

    public void setEmployed(boolean employed) {
        isEmployed = employed;
    }
    //endregion

}
